src/leafletControl.js

```diff
@@ -100,7 +100,8 @@ const Control = {
       const { form } = this.searchElement.elements;
       const root = map.getContainer().querySelector('.leaflet-control-container');
 
-      const container = createElement('div', 'leaflet-control-geosearch bar');
+      const container = L.DomUtil.create('div', 'leaflet-control-geosearch bar');
```

Comment:
I don't like the dependency on `L.DomUtil` here, we should use the `createElement` method as used on other places in the code.